src/Microsoft.Identity.Abstractions/ApplicationOptions/CredentialDescription.cs

```diff
@@ -136,6 +136,22 @@ public class CredentialDescription
         /// </summary>
         public string? SignedAssertionFileDiskPath { get; set; }
 
+        /// <summary>
+        /// Tenant in which the application is registered. This property is only used when
+        /// <see cref="SourceType"/> is <see cref="CredentialSource.AutoDecryptKeys"/>, in order
+        /// to determine the authority to use to get a token to get the decrypt the keys. The cloud
+        /// instance will be the same as the application, but the application can be a multi-tenant
+        /// application (tenant = <b>common</b> or <b>organizations</b>), and in this case to get a token, the
+        /// credential type needs to provide a tenant.
+        /// </summary>
+        public string? DecryptKeysApplicationTenant { get; set; }
+
+        /// <summary>
```

Comment:
Mention in doc this defaults to bearer?

Reply:
Hmm this comment popped up above where I meant it to. This is in reference to DecryptKeysProtocol property header.

Reply:
> Mention in doc this defaults to bearer?

Addressed (in the right comment, lol!)